docs/pages/contributor_guidelines.rst

```diff
@@ -41,3 +41,32 @@ Documentation
   'reddit.com'. This helps prevent any confusion between what is universally
   true between all reddits and what specifically applies to the most known
   instance.
+
+EOL Issues
+----------
+
+Depending on the OS you are editing the source on, you may run into one of two
+strange errors. Firstly, PRAW was written on a Unix-based OS. This means, that
+the source's ``EOLs`` (End Of Lines) are the Unix ``LF`` character, a.k.a "\\n"
+a.k.a hex ``0x0A``.
+
+On Windows, ``EOLs`` are ``CRLF``, which is ``0x0D0A`` ("\\r\\n") or if you are
+on a Mac with OS-9 and earlier ``EOLs`` are ``0x0D`` ("\\r"). More info on
+``EOLs`` can be found `here <http://bit.ly/1Ug8Xyo>`_.
+
+This can potentially pose one of two problems.
+
+1. Your ``EOLs`` may not be automatically converted and your code will break PRAW.
+
+2. Your editor automatically converts `EOLs`, at the cost that three invisible
+   hexadecimal values are added at the beginning of a line, which also breaks
+   PRAW. This can be determined to be the case if in the commit history on Github,
+   the a line of a file is removed and re-added with no visible change.
+
+The solutions to each are as follows:
```

Comment:
@bboe I don't have an example commit onhand, but this was a problem before. 

Also, some software such as my editor (Visual Studio Community) detects line endings that don't match that of the original file, and gives a prompt to convert them.

There are also cases in which sometimes the line endings are auto converted, but then there is garbage at the beginning of the file that is invisible such as in 445364e , which actually was the issue with my previous PR and all of the reverts